spec/unit/functions/hiera_spec.rb

```diff
@@ -121,5 +122,12 @@
       hiera_include.expects(:call_function_with_scope).with(scope,'include', %w[key foo])
       hiera_include.call(scope, 'foo') { |k| ['key', k] }
     end
+
+    it 'should do nothing when called during environment compilation' do
```

Comment:
This is not a unit test of the hiera_include function - should be moved to the spec for environment_compiler